src/Microsoft.SqlTools.ServiceLayer/DisasterRecovery/DisasterRecoveryService.cs

```diff
@@ -64,27 +61,49 @@ internal static ConnectionService ConnectionServiceInstance
         /// </summary>
         public void InitializeService(ServiceHost serviceHost)
         {
+            // Get database info
+            serviceHost.SetRequestHandler(ExtendedDatabaseInfoRequest.Type, HandleExtendedDatabaseInfoRequest);
```

Comment:
>ExtendedDatabaseInfoRequest [](start = 42, length = 27)

this message probably should be in the admin service with the default database info lookup code